### ISSUE TITLE
Loss 파일 제출 양식 관련 질문

### ISSUE THREAD

**Issue by hansmaan** [LISTatSNU/FastMRI_challenge#370]
안녕하세요. Room 218 팀의 한창희입니다.

Final Submission 시에, val_loss_npy.log 파일을 내야 하는데, Baseline 코드 상에서는 Validation Loss를 epoch 별로 작성하도록 되어 있습니다.

다만 저희 팀의 경우, No Validation으로 학습을 진행하고 있습니다.

이런 경우, val_loss_npy.log를 어떻게 제출드려야 할까요?

감사합니다.

**Comment by Racul** [comment]
@hansmaan 
조교 박정민입니다.

#340 
train log를 제출해주시고 readme에 작성해주시기 바랍니다.
